### PR TITLE
combine bootstrap css/js file for faster loading, create source maps for each

### PR DESCRIPTION
**Pull request by achen2401**
- enhance page loading time by combining all bootstrap js and css files into minify files respectively
- add sourcemap files for the js and css files for easy debugging in console/developer tool

### CODE REVIEW

File: portal/templates/layout.html [hunk 1]
```diff
@@ -9,23 +9,13 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta name="robots" content="noindex" />
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
```
Comment (ivan-c) [inline portal/templates/layout.html]: Would this version of [font awesome on cdnjs work](https://cdnjs.com/libraries/font-awesome/4.3.0)? Would be nice to move to a single host for libraries

Comment (achen2401) [inline portal/templates/layout.html]: Ivan, I missed that one, I think so, let me recreate source map to include that one.

Comment (achen2401) [inline portal/templates/layout.html]: @ivan-c 
Ivan, should be fixed.  Let me know if you see any problem.

**Comment by achen2401** [comment]
closing this per conversation with Ivan.

**Pull request closed without merging**